src/PipeMvc/dotnetCampus.Ipc.PipeMvcShare/HttpRequestMessageDeserializeContent.cs

```diff
@@ -21,12 +21,15 @@ public HttpRequestMessage ToHttpResponseMessage()
                 RequestUri = RequestUri,
             };
 
-            var memoryStream = new MemoryStream(Convert.FromBase64String(ContentBase64));
-            //var text = Encoding.UTF8.GetString(memoryStream.ToArray());
-            var streamContent = new StreamContent(memoryStream);
-            result.Content = streamContent;
+            if(ContentBase64 is not null)
```

Comment:
考虑到线程安全性，你应该使用此判断：

```suggestion
            if(ContentBase64 is { } contentBase64)
```

然后后续使用局部变量。

Reply:
@walterlv  无多线程问题，忽略